src/server/scripts/Northrend/Naxxramas/boss_faerlina.cpp

```diff
@@ -90,11 +91,12 @@ class boss_faerlina : public CreatureScript
         void EnterCombat(Unit * who)
         {
             BossAI::EnterCombat(who);
+            me->CallForHelp(VISIBLE_RANGE);
             summons.DoZoneInCombat();
             Talk(SAY_AGGRO);
             events.ScheduleEvent(EVENT_SPELL_POISON_BOLT, urand(12000,15000));
             events.ScheduleEvent(EVENT_SPELL_RAIN_OF_FIRE, urand(6000,18000));
-            events.ScheduleEvent(EVENT_SPELL_FRENZY, urand(60000,80000), 1);
+            events.ScheduleEvent(EVENT_SPELL_FRENZY, frenzyCooldown, 1);
```

Comment:
http://wowwiki.wikia.com/wiki/Grand_Widow_Faerlina
This will be used approximately every 60-80 seconds. It will increase Grand Widow Faerlina's physical damage by 150%, her melee haste by 50% and her size by 50%.

Reply:
I'll change it back and revert Patchwerk